src/main/java/net/sf/rails/algorithms/RevenueBonusTemplate.java

```diff
@@ -64,24 +64,24 @@ public void configureFromXML(Tag tag) throws ConfigurationException {
         if (trainTags != null) {
             for (Tag trainTag:trainTags) {
                 String type = trainTag.getAttributeAsString("type");
-                if (type != null) { 
+                if (type != null) {
                     identTrainTypes.add(type);
                 }
             }
         }
 
-        // check for phases 
+        // check for phases
         List<Tag> phaseTags = tag.getChildren("phase");
         if (phaseTags != null) {
             for (Tag phaseTag:phaseTags) {
                 String type = phaseTag.getAttributeAsString("name");
-                if (type != null) { 
+                if (type != null) {
                     identPhases.add(type);
                 }
             }
         }
-        log.info("Configured " + this);
-        
+        log.debug("Configured {}", this);
+
```

Comment:
Why does it change the log level?

Reply:
Because this level of logging detail should match what the rest (or most) of the application is using for log levels. Most of the code is using debug for the lower level, developer useful information (as well it should) and info/warn/error for information that would be helpful for a non-developer. The detail being logged here seemed to fall into the category of information more relevant to a developer reviewing log data than to an end user and hence the level change. There are some pretty consistent standards on use of log levels and while the bulk of the code base does seem to follow them there was a chunk of code that seemed out of place to what the rest of the code was doing-- the bulk of that was this level of `info` to `debug`. So again, while there are standards, in this case it was more about bringing about consistency (or following existing consistency) within the code base.

Reply:
:+1: